.gitignore

```diff
@@ -12,3 +12,5 @@ Thumbs.db
 *.class
 *.idea
 
+
+algorithms/Python/arrays/unique_character.py
```

Comment:
looks like you accidentally created this project somewhere else and didn't wanted git to track that particular file. 

it's recommended for `.gitignore` to only contain files which are unnecessary and could be generated when working with the project itself (for example: compiler generated files, project files etc. ). 

If you want some files to be untracked exclusively on your local git and don't want to pollute `.gitignore` then it's recommended to update `.git/info/exclude` instead.

Reply:
I will update shortly. Thank you for the review.